### PR TITLE
Add inline demo

### PR DESCRIPTION
**Pull request by samuelli**

### CODE REVIEW

**Comment by samuelli** [comment]
Preview link in travis checks

File: README.md [hunk 1]
```diff
@@ -1,8 +1,72 @@
...
+        }
+      }
+    </style>
+    <next-code-block></next-code-block>
+  </template>
+</custom-element-demo>
+```
+-->
+```html
+<app-header reveals>
```
Comment (blasten) [inline README.md]: this might need `@apply --layout-fit` in order for it to work correctly.

Comment (samuelli) [inline README.md]: It has `@apply(--layout-fixed-top)` above

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**